projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -872,6 +873,7 @@ public function get_filtered_connection_data( $selected_post_id = null ) {
 					'enabled'         => $enabled,
 					'done'            => $done,
 					'toggleable'      => $toggleable,
+					'follower_count'  => ! empty( $connection_data['meta']['follower_count'] ) ? $connection_data['meta']['follower_count'] : -1,
```

Comment:
The `empty()` causes 0 follower count to fall into -1. Should `isset()` be used over `empty()` to cover cases where users have 0 followers, e.g. if follower count not set, then -1.